supported-tools.md

```diff
@@ -291,6 +291,7 @@ formatting.
   * [nasm](https://www.nasm.us/) :floppy_disk:
 * Nim
   * [nim check](https://nim-lang.org/docs/nimc.html) :floppy_disk:
+  * [nimlsp](https://github.com/PMunch/nimlsp)
```

Comment:
The CI job will tell you to update the similar Vim help file next, `doc/ale-supported-languages-and-tools.txt`.

Reply:
Right, thanks!